<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;MV Boli&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="TNotOpenWidget" name="notOpenWidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="openPicture"/>
    <addaction name="savePicture"/>
    <addaction name="saveAsPicture"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetFloatable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::LeftDockWidgetArea|Qt::DockWidgetArea::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="okButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="cancelButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="resetButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>重置参数</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QScrollArea" name="centralScrollArea">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>156</width>
          <height>486</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#scrollAreaWidgetContents {
	border: 1px solid rgb(0, 64, 64);
	border-radius: 15px;;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::ToolBarArea::BottomToolBarArea|Qt::ToolBarArea::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fallback"/>
   <addaction name="forward"/>
   <addaction name="separator"/>
   <addaction name="widthAlignment"/>
   <addaction name="heightAlignment"/>
   <addaction name="resizeAlignment"/>
   <addaction name="separator"/>
  </widget>
  <action name="openPicture">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="savePicture">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="saveAsPicture">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="widthAlignment">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/images/icons/MaterialSymbolsFitPageWidthOutlineRounded.svg</normaloff>:/MainWindow/images/icons/MaterialSymbolsFitPageWidthOutlineRounded.svg</iconset>
   </property>
   <property name="text">
    <string>宽对齐</string>
   </property>
   <property name="toolTip">
    <string>对齐宽度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="heightAlignment">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/images/icons/MaterialSymbolsFitPageHeightOutlineRounded.svg</normaloff>:/MainWindow/images/icons/MaterialSymbolsFitPageHeightOutlineRounded.svg</iconset>
   </property>
   <property name="text">
    <string>高对齐</string>
   </property>
   <property name="toolTip">
    <string>对齐高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="resizeAlignment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/images/icons/GardenOriginalSizeFill16.svg</normaloff>:/MainWindow/images/icons/GardenOriginalSizeFill16.svg</iconset>
   </property>
   <property name="text">
    <string>原大小</string>
   </property>
   <property name="toolTip">
    <string>对齐原大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="fallback">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/images/icons/SolarMultipleForwardLeftBroken.svg</normaloff>:/MainWindow/images/icons/SolarMultipleForwardLeftBroken.svg</iconset>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
   <property name="toolTip">
    <string>向后退一步</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="forward">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/images/icons/SolarMultipleForwardRightBroken.svg</normaloff>:/MainWindow/images/icons/SolarMultipleForwardRightBroken.svg</iconset>
   </property>
   <property name="text">
    <string>前进</string>
   </property>
   <property name="toolTip">
    <string>向前进一步</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="closeImage">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/images/icons/MaterialSymbolsCloseRounded.svg</normaloff>:/MainWindow/images/icons/MaterialSymbolsCloseRounded.svg</iconset>
   </property>
   <property name="text">
    <string>关闭图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="toolBarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="parameterAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>参数窗口</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TNotOpenWidget</class>
   <extends>QWidget</extends>
   <header>tnotopenwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
